lib/logstash/outputs/email.rb

```diff
@@ -80,6 +80,9 @@ class LogStash::Outputs::Email < LogStash::Outputs::Base
   # Body for the email - plain text only.
   config :body, :validate => :string, :default => ""
 
+  # Email template file to be used - as mustache template.
+  config :template_file, :validate => :string, :default => ""
```

Comment:
we can use `:validate => :path` here instead, so we can check the file actually exists

Reply:
I had to remove the default and therefore changed the `.empty?` to `nil?`